configure.py

```diff
@@ -1675,6 +1675,8 @@ def configure_v8(o, configs):
     raise Exception(
         'Only one of the --v8-enable-object-print or --v8-disable-object-print options '
         'can be specified at a time.')
+  if o['variables']['v8_enable_webassembly'] and o['variables']['target_arch'] in ('x64', 'x86_64'):
```

Comment:
```suggestion
  if o['variables']['v8_enable_webassembly'] and o['variables']['target_arch'] == 'x64':
```

`target_arch` is normalized before being assigned to `o['variables']`: 
https://github.com/yolanda15/node/blob/07ba285fa3118f40c252284287fd788d95fa5ad2/configure.py#L1404-L1405

Reply:
Thanks for the suggestion! Updated in the new commit. Please take a look, Thanks!